src/css/base.scss

```diff
@@ -32,81 +34,110 @@ header {
       align-self: flex-end;
     }
   }
-}
-
-img {
-  height: 100px;
+  input {
+    font-family: $bodyfont;
+    border: none;
+    height: 3vh;
+    background-color: $lightc;
```

Comment:
I might look to change these so be more descriptive for the colors

Reply:
Totally, my thought process was having them be a cool and warm hue of whatever value they occupy, but thats a good thought for my refactor and comb over now that they are less deisgn place holders and more actual decisions.